### ISSUE TITLE
Message Type: Error

### ISSUE THREAD

**Issue by etkirsch** [neuravion/mesh-chat#29]
A way we might combat the infinite nodelist loop is to provide error messages to clients. These are, of course, encrypted. It also would provide the possibility of auto-sending an encryption failed error message to another client.

Any thoughts?

**Comment by NullVoxPopuli** [comment]
I think this is a good idea. Though, we should avoid mandatory encryption failed messages, cause those could end in infinite sending as well.

**Comment by etkirsch** [comment]
Of course. I think we should have a small set of _enforced_ errors, but the rest would be left to the client.
